src/assets/index.js

```diff
@@ -0,0 +1,18 @@
+export {default as music1} from './music/music1.mp3'
```

Comment:
music is dynamic files. don't store it in assets

Reply:
Where should this store?